src/sparsezoo/v2/objects/model.py

```diff
@@ -215,23 +207,36 @@ def generate_outputs(
             yield output
 
     def download(
```

Comment:
Thoughts on making this the main function for getting the model path in place of `get_path`? This could just return the model path instead of a bool:

```diff
-model.download(override=True)
-model_path = model.get_path()
+model_path = model.download(override=True)
```

Reply:
Actually following up with this - could users just do `Model("zoo:...", download=True)`? so its just one line for them?

Reply:
The first suggestion is very interesting, I like it. What do you think @bfineran ?
The second suggestion is probably not what I'd like to achieve. We are pretty convinced that `Model`'s `__init__` should be a "monadic" method. The user should just enter the `path` to the model and not care about any additional flags.